### PR TITLE
Add react-mount module to related modules. #506

### PR DESCRIPTION
**Pull request by greenish**

### CODE REVIEW

**Comment by AlexKVal** [comment]
Just for references https://github.com/react-bootstrap/react-bootstrap/issues/506

**Comment by AlexKVal** [comment]
To compare https://github.com/PixelsCommander/ReactiveElements

**Comment by mtscout6** [comment]
@AlexKVal Good find.

@greenish Looks like the ReactiveElements repo has garnered more support. I don't want to embark on a holy war of any kind, but what does your library provide over ReactiveElements? Should we reference both?

**Comment by greenish** [comment]
Thanks for the response – Good find indeed! 

I had a look at the ReactiveElements project. ReactiveElements introduces its own custom API within the "mounted" react component. For React-mount I took great care to enable all the "native" react features without the need to rewrite components. This makes react-mount a small layer between html and any react component (not a library at all) and makes it possible to mount all react-bootstrap components without modifying them. This is not possible with the ReactiveElement i.e. all nested elements would need a specific rewrite. A ReactiveElement has to be specifically written as a ReactiveElement.
As for community support, I barely wrote the module three weeks ago and haven't been advertising it at all apart from this entry.

TL;DR React-boostrap doesn't work with ReactiveElements out of the box.

best,
Philipp

**Comment by AlexKVal** [comment]
Good points. 
It seems your project has an advantage :sparkles: 

I like the idea that there is no need to modify existing components. It's a win-win :sparkle: :sparkle: 
It theory there will be more projects using `react-bootstrap` with `react-mount` help.

I propose to create `Complementary-tools` page on the Wiki and put there links with author's description.
The comparative analysis written in previous post is a good candidate, I think :wink:

**Comment by mtscout6** [comment]
@greenish I like @AlexKVal's recommendation to link your project from the complementary tools page in the React wiki. Since your library can be used with more libraries than just react-bootstrap, that would be a more appropriate place to advertise your project.

**Comment by greenish** [comment]
@AlexKVal I agree, It's a win-win. And people using react-boostrap should know about the possibility. React-mount makes react-boostrap (and react based front end libraries in general) more viable for users not using react.
From a UX perspective the mounted react-boostrap is even easier to use and more user-friendly than the _real_ Bootstrap ever was:

```html
  <body>
    <Carousel>
      <CarouselItem>
        <img width={1140} height={500} alt='1140x500' src='http://placekitten.com/g/1140/500'/>
        <div class='carousel-caption'>
          <h3>First slide label</h3>
          <p>Nulla vitae elit libero, a pharetra augue mollis interdum.</p>
        </div>
      </CarouselItem>
      <CarouselItem>
        <img width={1140} height={500} alt='1140x500' src='http://placehold.it/1140x500'/>
        <div class='carousel-caption'>
          <h3>Second slide label</h3>
          <p>Nulla vitae elit libero, a pharetra augue mollis interdum.</p>
        </div>
      </CarouselItem>
      <CarouselItem>
        <img width={1140} height={500} alt='1140x500' src='http://placehold.it/1140x500/ff00ff/ffff00'/>
        <div class='carousel-caption'>
          <h3>Third slide label</h3>
          <p>Nulla vitae elit libero, a pharetra augue mollis interdum.</p>
        </div>
      </CarouselItem>
    </Carousel>

    <!-- make complete react-boostrap available -->
    <script src="mounted-boostrap.js"></script>

  </body>
```
(From: https://github.com/greenish/react-mount/tree/master/examples/react-bootstrap-browserify)

@mtscout6 The facebook complementary tools page is extensive. None will be able to find the module and make a connection to react-bootstrap. I will still send a pull-request to them, but this will not help react-bootstrap in any way.
So I would still suggest to merge this pull-request or even ship react-mount with the bower version and show an example on the getting started page. But thats all up to you, I made my case – if you're not interested, I'll leave it at that.

best
Philipp
